release/models/platform/openconfig-platform.yang

```diff
@@ -403,12 +409,51 @@ module openconfig-platform {
 
     leaf location {
       type string;
+      status deprecated;
       description
         "System-supplied description of the location of the
         component within the system. This could be a bay position,
         slot number, socket location, etc. For component types that
         have an explicit slot-id attribute, such as linecards, the
-        system should populate the more specific slot-id.";
+        system should populate the more specific slot-id.
+
+        This leaf is deprecated and replaced by parent-position.";
```

Comment:
We should update this to also say `install-position` instead of `parent-position`. Same applies to the version description in release/models/platform/openconfig-platform-common.yang.